src/pcb/pcb_board.ts

```diff
@@ -32,12 +57,19 @@ export interface PcbBoard {
   subcircuit_id?: string
   width?: Length
   height?: Length
+  display_offset_x?: string
+  display_offset_y?: string
   thickness: Length
   num_layers: number
   center: Point
   outline?: Point[]
   shape?: "rect" | "polygon"
   material: "fr4" | "fr1"
+  anchor_position?: Point
+  anchor_alignment: NinePointAnchor
+  position_mode?: "relative_to_panel_anchor" | "grid_in_panel" | "none"
```

Comment:
i think this is a good idea, but i think for simplicity in the short term we just make it all relative_to_panel_anchor

```suggestion
  position_mode?: "relative_to_panel_anchor" | "none"
```